docs/resources/offload.md

```diff
@@ -3,12 +3,16 @@
 page_title: "liqo_offload Resource - liqo"
 subcategory: ""
 description: |-
-  Offload a namespace.
+  Once a given namespace is selected for offloading, Offload resource extends
```

Comment:
The word "Offload resource" is difficult to read in the text. Since it refers to the name of a resource, should we write it in Italic on in 'code' style to make things more clear?
E.g., Once a given namespace is selected for offloading, `Offload resource` extends
(if you agree, please propagate this change across all the documentation)